docs/2_1_common_math.adoc

```diff
@@ -10,10 +10,10 @@ FMU and importer use variables to exchange information.
 The properties and semantics of variables are described in the <<modelDescription.xml>>.
 Access to variable values is possible via appropriate API functions.
 
-The <<independent>> variable latexmath:[\mathbf{t} \in \mathbb{T}] _[typically: time]_ is a tuple latexmath:[\mathbf{t} = (\mathbf{t}_R,\mathbf{t}_I)], where latexmath:[\mathbf{t}_R \in \mathbb{R},\ \mathbf{t}_{I} \in \mathbb{N} = \{0, 1, 2, \ldots\}].
-The real part latexmath:[\mathbf{t}_R] of this tuple is the <<independent>> variable of the FMU for describing the continuous-time behavior of the model between events.
-During continuous-time integration the integer part of time latexmath:[\mathbf{t}_I = 0].
-latexmath:[\mathbf{t}_I] is a counter to enumerate (and therefore distinguish) the events at the same continuous-time instant latexmath:[\mathbf{t}_R].
+The <<independent>> variable latexmath:[t \in \mathbb{T}] _[typically: time]_ is a tuple latexmath:[t = (t_R,t_I)], where latexmath:[t_R \in \mathbb{R},\ t_{I} \in \mathbb{N} = \{0, 1, 2, \ldots\}].
```

Comment:
t_I does not belong to the natural number subscript I does->
change
t_{I} \in \mathbb{N}
to I \in \mathbb{N}

Reply:
Same as above...